src/wrappers/batch.js

```diff
@@ -11,6 +12,7 @@ const batchRunner = require('../runners/aws_batch.js');
  * Epsagon's batch Job wrapper, sets tracing on a batch job
  */
 module.exports.wrapBatchJob = function wrapBatchJob() {
+    tracer.traceGetter = traceObject.get;
```

Comment:
why not keep the name? What I would do is make traceGetter have the traceObject/traceContext object, which both implement a `get` interface. then to invoke the events / tracer would use `traceGetter.get()`

Reply:
also i would have a traceNoop which does nothing, and that's what the tracer is default value is (initialized with it) so we won't have errors like the ones we had